skore/src/skore/sklearn/_comparison/metrics_accessor.py

```diff
@@ -112,7 +112,7 @@ def report_metrics(
         aggregate : {"mean", "std"}, list of such str or None, default=("mean", "std")
             Function to aggregate the scores across the cross-validation splits.
             None will return the scores for each split.
-            Ignored when comparison is between :class:`~skore.EstimatorReport`s.
+            Ignored when comparison is between :class:`~skore.EstimatorReport(s)`.
```

Comment:
IMO, you should change for:

```suggestion
            Ignored when comparison is between :class:`~skore.EstimatorReport` instances.
```

This is better than `~skore.EstimatorReport reports` which is redundant, or `~skore.EstimatorReport(s)` which is confusing to read in raw text.

Reply:
That's a good suggestion @thomass-dev 😀. I was torn between using entries, items or objects but instances is great. I'll adjust for both files.